ui/pages/institutional/custody/index.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../components/institutional/jwt-dropdown/jwt-dropdown.scss';
+@import '../../../components/institutional/jwt-url-form/jwt-url-form.scss';
+
+.custody-select-all {
+  font-size: 12px;
+}
```

Comment:
Use Text Component variant sizing not font size css

Reply:
will do 👍🏼